requirements.yml

```diff
@@ -0,0 +1,4 @@
+---
+collections:
+  - community.general
+  - ansible.posix
```

Comment:
This is used in #105 so I added it in advance.

Reply:
Thank you